jetbrains-core/src/software/aws/toolkits/jetbrains/services/caws/envclient/CawsEnvironmentClient.kt

```diff
@@ -119,9 +126,21 @@ class CawsEnvironmentClient(
             val httpRequest = HttpPut("$endpoint/activity").also {
                 it.entity = StringEntity(body, ContentType.APPLICATION_JSON)
             }
-            val response = execute(httpRequest).use {}
+
+            execute(httpRequest).use {
+                // defensive in case they decide to actually start returning 200s
+                if (it.statusLine.statusCode != 200) {
+                    val response = objectMapper.readValue<Error>(it.entity.content)
+                    LOG.debug { "Wrote '$request' to /activity with error response: $response" }
```

Comment:
Should this be marked as a an error in the logs since the request wasn't actually successful?